src/settings/BatchGroupConfigurationSettings/BatchGroupConfigurationSettings.js

```diff
@@ -45,6 +49,24 @@ const BatchGroupConfigurationSettings = ({ mutator }) => {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   []);
 
+  const fetchBatchVoucherExports = useCallback(
```

Comment:
you don't  need useCallback here

Reply:
why not? it's just to avoid creation of new function on every render